tests/tokenizers.test.js

```diff
@@ -33,3 +33,13 @@ describe('Tokenizers', () => {
         });
     }
 });
+
+describe('Edge cases', () => {
+    it('should not crash when encoding a very long string', async () => {
+        let tokenizer = await AutoTokenizer.from_pretrained('t5-small');
+
+        let text = String.prototype.repeat.call('Hello world! ', 50000);
```

Comment:
`'Hello world! '.repeat(50000);` :pinched_fingers: 

Kinda sad to see the nice array spreading syntax gone, but this is the reality of large arrays, nice work :1st_place_medal: